app/retail/templates/about.html

```diff
@@ -19,6 +19,11 @@
 <html lang="en">
   {% include 'shared/head.html' %}
   <head>
+    {% block title %}
+      About Gitcoin: Find Open Bug Bounties & Freelance Coders
```

Comment:
Can you use `{% trans "" %}` for all the titles? Else the title doesn't get translated and since it is a visible component, it needs to be translated.